### DISCUSSION TITLE
NOAA NBM cfgrib issue fixed?

### DISCUSSION THREAD

**Discussion by v7nguyen** [blaylockbk/Herbie#88] [General]
Hi Brian,

I came across a post you submitted on the ECMWF GitHub regarding NBM grib file lines being written backwards: https://github.com/ecmwf/cfgrib/issues/240

I used your sample code and am still getting the strange results even with eccodes v2.25.0. Are you experiencing this behavior as well or is it fixed for you?

![image](https://user-images.githubusercontent.com/53784737/181081723-d9f7beaf-88e4-4677-8037-04265a4b03c0.png)

**Comment by blaylockbk** [comment]
Yeah, I haven't found a workaround for this using cfgrib. I think it'll work reading with pygrib, but haven't tried it.

**Comment by blaylockbk** [comment]
If you still see this as a problem, reopen that issue in cfgrib's repo.
